### PR TITLE
Prepare for 1.8.0-rc1 release

### PR DESCRIPTION
**Pull request by warrenvw**
Prepare `release-1.8` branch for the `1.8.0-rc1` release.

### CODE REVIEW

File: hack/catalog.sh [hunk 1]
```diff
@@ -30,8 +30,8 @@ elif [ -n "$DOCKER_REPO_OVERRIDE" ]; then
   export IMAGE_KNATIVE_OPERATOR="${DOCKER_REPO_OVERRIDE}/knative-operator"
   export IMAGE_KNATIVE_OPENSHIFT_INGRESS="${DOCKER_REPO_OVERRIDE}/knative-openshift-ingress"
 else
-  export IMAGE_KNATIVE_OPERATOR="registry.svc.ci.openshift.org/openshift/openshift-serverless-nightly:knative-operator"
-  export IMAGE_KNATIVE_OPENSHIFT_INGRESS="registry.svc.ci.openshift.org/openshift/openshift-serverless-nightly:knative-openshift-ingress"
+  export IMAGE_KNATIVE_OPERATOR="registry.svc.ci.openshift.org/openshift/openshift-serverless-v1.8.0-rc1:knative-operator"
```
Comment (matzew) [inline hack/catalog.sh:33]: I will add a CI job for exactly this promotion

**Comment by mgencur** [comment]
AFAICT, this looks good.

File: hack/lib/vars.bash [hunk 2]
```diff
@@ -48,7 +48,7 @@ readonly UPGRADE_CLUSTER="${UPGRADE_CLUSTER:-"false"}"
 readonly UPGRADE_OCP_IMAGE="${UPGRADE_OCP_IMAGE:-}"
 
 readonly INSTALL_PREVIOUS_VERSION="${INSTALL_PREVIOUS_VERSION:-"false"}"
-export OLM_CHANNEL="${OLM_CHANNEL:-"preview-4.6"}"
+export OLM_CHANNEL="${OLM_CHANNEL:-"4.3"}"
```
Comment (markusthoemmes) [inline hack/lib/vars.bash:51]: This butchered CI I believe, as we'll now no longer install 1.8 in CI leading to a battery of errors. it even seems like the `-rc1` format isn't supported by OLM.

**Pull request merged**